feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -364,47 +386,52 @@ func TestVrfPolicy(t *testing.T) {
 
 	// Configure DUT interfaces and forwarding policy
 	configureDUT(t, dut, p1, p2)
+	configdefaultRoute(t, dut, ATEDESTIPv4VLAN20, defaultNHv4, ATEDESTIPv6, defaultNHv6)
+	configvrfRoute(t, dut, ATEDESTIPv4VLAN10, vrfNH)
 
 	// Configure ATE
 	ate := ondatra.ATE(t, "ate")
-	allFlows := configureATE(t, ate)
+	configureATE(t, ate)
+
+	allFlows := createTraffic(t, ate)
 
 	tcs := []struct {
-		desc      string
-		policy    string
-		passFlows []string
+		desc           string
+		policy         string
+		endpoints      []string
+		passFlowsnames []string
+		passFlows      []*ondatra.Flow
```

Comment:
These don't look like flows you expect to "pass" but all the flows for which you are sending traffic. I'd just call it "flows"

Reply:
ok. will rename this variable name to stream(s).

Reply:
Done in new commit.